src/language/codegen.yaml

```diff
@@ -87,7 +87,30 @@
                                   type: StatementBlock
                               - finalize_block:
                                   brief: "Statement block to be executed after calling linear solver"
-                                  type: StatementBlock                                  
+                                  type: StatementBlock
+                        - DerivativeOriginalFunctionBlock:
```

Comment:
Why the `Function` in `DerivativeOriginalFunctionBlock`?

Reply:
NMODL doesn't allow two blocks with the same type (doesn't matter that they may not have the same name), and we will need another block for the transformation involving the Jacobian, so it's basically with that in mind that there's the additional `Function`. Open to suggestions on how to make it shorter though!